tests/chainermn_tests/extensions_tests/test_allreduce_persistent.py

```diff
@@ -20,7 +20,15 @@ def __init__(self, n_in=3, n_units=5, n_out=2):
 
 class TestAllreducePersistent(unittest.TestCase):
 
-    def _test(self, comm, model):
+    def _test(self, comm, model, use_gpu, use_chx):
+        if use_gpu:
+            # Use CuPy's Device class to force call cudaSetDevice()
+            chainer.cuda.get_device_from_id(comm.intra_rank).use()
+
+        device = get_device(comm.intra_rank if use_gpu else None, use_chx)
+        device.use()
```

Comment:
Unnecessary

Reply:
I'm not sure this is unnecessary. CUDA's current GPU and ChainerX's current GPU are different. I'll consult ChainerX developers.

Reply:
As we discussed, this code is unnecessary, the succeeding `to_device` serve the purpose.

Reply:
I'll fix it.